tools/hess/hess_astro_tool.xml

```diff
@@ -0,0 +1,138 @@
+<tool id="hess_astro_tool" name="hess_astro_tool" version="0.0.1+galaxy0" profile="23.0">
+  <requirements>
+    <requirement type="package" version="8.18.1">ipython</requirement>
+    <requirement type="package" version="6.0.0">astropy</requirement>
+    <requirement type="package" version="1.1">gammapy</requirement>
+    <requirement type="package" version="3.8.2">matplotlib</requirement>
+    <requirement type="package" version="0.4.6">astroquery</requirement>
+    <requirement type="package" version="1.11.4">scipy</requirement>
+    <requirement type="package" version="1.2.3">oda-api</requirement>
+    <!--Requirements string 'nb2workflow[cwl,service,rdf,mmoda]>=1.3.30 
+' can't be converted automatically. Please add the galaxy/conda requirement manually or modify the requirements file!-->
+    <requirement type="package" version="7.13.0">nbconvert</requirement>
+    <requirement type="package" version="1.20.3">wget</requirement>
+  </requirements>
+  <command detect_errors="exit_code">ipython '$__tool_directory__/${_data_product._selector}.py'</command>
```

Comment:
why ipython?

Reply:
As far as this is converted from notebook, which can contain some jupyter magic, using ipython to run the script assures they will be properly treated. It's not a great sacrifice to have an additional requirement, but the logic may be improved a bit to use it only when magic commands are actually there